obspy/clients/fdsn/mass_downloader/utils.py

```diff
@@ -102,12 +102,15 @@ def download_and_split_mseed_bulk(client, client_name, chunks, logger):
     # intervals, each of which will end up in a separate file.
     filenames = collections.defaultdict(list)
     for chunk in chunks:
-        filenames[tuple(chunk[:4])].append({
+        key = tuple(chunk[:4])
+        filename = {
             "starttime": chunk[4],
             "endtime": chunk[5],
             "filename": chunk[6],
             "current_latest_endtime": None,
-            "sequence_number": None})
+            "sequence_number": None}
+        if filename not in filenames[key]:
+            filenames[key].append(filename)
```

Comment:
Without debugging into this, according to the docstring `chunks` is a list of tuples, so you should be able to get rid of duplicates by just doing:

```python
chunks = list(set(chunks))
```

Reply:
Thanks @megies. That's the first thing I tried, but those tuples are not hashable, so it's not possible to use `set()`.

Reply:
Oh, OK then.